### PR TITLE
doc: Add changelog for 0.15

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Comment by temanbrcom** [comment]
@grianbrcom

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.15.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.14.0...0.15.0) (2020-09-30)
+
+### Bug Fixes
+* Correct coloring for CICS statements [#469](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/469)
+* Disable the spellchecker for variables [#478](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/478)
+
+### Features
+* Add syntax analysis for CICS statements ([#469](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/469))
```
Comment (grianbrcom) [inline CHANGELOG.md:11]: extra parentheses around a link

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
 
+## [0.15.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.14.0...0.15.0) (2020-09-30)
+
+#### Added
+- Syntax analysis for CICS statements
+- Telemetry event recording
+- CICS variables support
+- Locking copybook downloading queue in case of wrong mainframe credentials
+
+#### Changed
```
Comment (grianbrcom) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md:12]: Nice idea about using `changed` section!

**Review by grianbrcom — APPROVED** [review]
:octocat:

File: CHANGELOG.md (hunk 1 above)
Comment (temanbrcom) [reply CHANGELOG.md]: 👌

**Review by zeibura — APPROVED** [review]
I moved the copybook queue lock from "added" to "changed", hope that's okay. Otherwise fine by me.

**Comment by temanbrcom** [comment]
Makes senses. Thanks, Zeb

**Pull request merged**